internal/server/kong/ws/config/compat/compatibility_table.go

```diff
@@ -834,6 +842,20 @@ var (
 				},
 			},
 		},
+		{
+			Metadata: config.ChangeMetadata{
+				ID:          config.ChangeID("P135"),
+				Severity:    config.ChangeSeverityError,
+				Description: standardCoreEntityMessage("vault", "3.0"),
+				Resolution:  standardUpgradeMessage("3.0"),
+			},
+			SemverRange: versionsPre300,
+			Update: config.ConfigTableUpdates{
+				Name:   "vault",
```

Comment:
```suggestion
				Name:   config.Vault.String(),
```
nit: for consistency with other changes